circuit/algorithms/src/bhp/hash_uncompressed.rs

```diff
@@ -80,9 +80,11 @@ mod tests {
             let native = console::$bhp::<<Circuit as Environment>::Network>::setup(DOMAIN)?;
             let circuit = $bhp::<Circuit>::constant(native.clone());
 
+            let mut rng = TestRng::default();
```

Comment:
The macro wrapping this call is called multiple times in test cases lower down, thus creating the rng multiple times per test.